File: cpp/zj-d072.cpp

#include <iostream>
using namespace std;

int main() {
    int y, n;
    
    cin >> n;
    for (int TC = 1; TC <= n; TC++) {
        cin >> y;
        cout << "Case " << TC << ": ";
        if ((y % 4 == 0 && y % 100 != 0) || (y % 400) == 0)
            cout << "a leap year\n";
        else
            cout << "a normal year\n";
    }
    return 0;
}

File: python/zj-d072.py
#!/usr/bin/env python 

n = int(input())
for TC in range(1, n+1):
    print(f"Case {TC}: ", end="")
    y = int(input())
    if ((y % 4 == 0) and (y % 100 != 0)) or (y % 400 == 0):
        print("a leap year")
    else:
        print("a normal year")
